### PR TITLE
Update tags to pick RC2

### PR DESCRIPTION
**Pull request by SenRamakri**
Release candidate 2 for 2.0.0 release to include maestro updates for config management.

### CODE REVIEW

**Comment by SenRamakri** [comment]
I'll draft RC2 release when the below PRs are reviewed and merged:
https://github.com/armPelionEdge/manifest-pelion-edge/pull/21
https://github.com/armPelionEdge/meta-pelion-edge/pull/96

File: default.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <!-- build scripts -->
     <project name="armpelionedge/build-pelion-edge"
         path="build-env"
-        revision="refs/tags/2.0.0-rc1"
+        revision="refs/tags/2.0.0-rc2"
```
Comment (theotherjimmy) [inline default.xml]: You have not made any changes to this repo, so it does not need an RC2

Comment (SenRamakri) [reply default.xml]: Well, if we can move the RC1 tag on meta-pelion-edge, we don't need RC2. Is that what you are asking?

Comment (theotherjimmy) [reply default.xml]: This is `build-pelion-edge`

**Comment by SenRamakri** [comment]
@cmonr @x85446 - @theotherjimmy and I talked about this and we think its better to move RC1 tag in meta-pelion-edge to capture maestro changes as it hasn't been long since we released RC1. So, I'm going to close this PR. Once meta-peliion-edge PR(https://github.com/armPelionEdge/meta-pelion-edge/pull/96) is merged I'll update the release notes.

**Pull request closed without merging**